api/app/src/routes/user.ts

```diff
@@ -115,6 +117,51 @@ router.get(
   })
 );
 
+/** ---------------------------------------------------------------------------------------
+ * DELETE /user/revoke
+ *
+ * Revoke access to a provider
+ *
+ * @param   {string}  req.query.provider    The provider to revoke access.
+ * @param   {string}  req.query.userId      The internal user ID.
+
+ * @return  {{success: boolean}}      If successfully removed.
+ */
+router.delete(
+  "/revoke",
```

Comment:
This is not the same token we generate on `/connect/token`, right?

If it is, we could `delete` on the same endpoint for consistency. Disregard if its not, please :)